coordinator/tests/services/translation/request/dispense/dispense-claim.spec.ts

```diff
@@ -1,24 +1,22 @@
 import * as TestResources from "../../../../resources/test-resources"
-import requireActual = jest.requireActual
 import {MomentFormatSpecification, MomentInput} from "moment"
 import {fhir, hl7V3} from "@models"
 import {toArray} from "../../../../../src/services/translation/common"
 import {clone} from "../../../../resources/test-helpers"
-import pino = require("pino")
 import {convertDispenseClaim} from "../../../../../src/services/translation/request/dispense/dispense-claim"
-import {
-  createAgentPersonFromAuthenticatedUserDetailsAndPractitionerRole
-} from "../../../../../src/services/translation/request/agent-unattended"
-
-const logger = pino()
+import * as testData from "../../../../resources/test-data"
+import requireActual = jest.requireActual
```

Comment:
Looks like we are using `requireActual` to basically mock dateTime, we could probably achieve the same for a lot less overhead using `jest.useFakeTimers`

No something we need to fix in this PR but I'd be a nice cleanup